scripts/kill-dev-processes.mjs

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env node
```

Comment:
Let's use TypeScript instead of JavaScript as the main language for writing node.js programs. JavaScript belongs only in tests that specifically verify compatibility with JavaScript. Agents can execute such migrations very quickly in my experience.